nixos/modules/programs/steam.nix

```diff
@@ -57,6 +72,14 @@ in {
         allowedUDPPorts = [ 27015 ]; # Gameplay traffic
       })
     ];
+
+    # Append the extra compatibility packages to whatever else the env variable was populated with.
+    # For more information see https://github.com/ValveSoftware/steam-for-linux/issues/6310#issuecomment-511630263
+    environment.sessionVariables = mkIf (cfg.extraCompatPackages != []) {
+      STEAM_EXTRA_COMPAT_TOOLS_PATHS = strings.makeBinPath cfg.extraCompatPackages;
+    };
+
+    environment.systemPackages = [ steam steam.run ] ++ cfg.extraCompatPackages;
```

Comment:
Is this necessary?

Reply:
I did update my module override to that quite some time ago, and this is indeed not necessary to get steam pick up the declared proton package, to on my opinion that can be removed safely.